Sources/UIAlertControllerView.swift

```diff
@@ -47,18 +51,17 @@ class UIAlertControllerView: UIView {
         layer.cornerRadius = 3
 
         header.sizeToFit()
-        header.frame.height += verticalPadding
         header.layer.contentsGravity = .top
         addSubview(header)
+        addSubview(text)
```

Comment:
We add the view to the hierarchy here - but in the `layoutAsActionSheet()` case - we don't layout this view.

Maybe we don't want to implement this at the moment - in which case we could consider adding an assertion when `style == . actionSheet`.

Alternatively we could render this. I'm not sure if the native android Dialogs do actually allow this.